### PR TITLE
[CDF-8746] remove parse

### PR DESCRIPTION
**Pull request by chingyu-cognite**

### CODE REVIEW

**Comment by rebwib** [comment]
Need to bump version and add to changelog. 
Also, I think we should add `convert` so that the get `svg_url` functionality is still in the SDK.

File: tests/tests_integration/test_contextualization/test_pnid_parsing.py [hunk 1]
```diff
@@ -1,15 +1,37 @@
 import pytest
 
 from cognite.experimental import CogniteClient
-from cognite.experimental.exceptions import ModelFailedException
+from cognite.experimental.data_classes import ContextualizationJob
 
 COGNITE_CLIENT = CogniteClient()
 PNIDAPI = COGNITE_CLIENT.pnid_parsing
+PNID_ID = 3261066797848581
```
Comment (rebwib) [inline tests/tests_integration/test_contextualization/test_pnid_parsing.py:8]: Maybe rename to `PNID_FILE_ID`?

**Comment by rebwib** [comment]
Looks good, you still need to bump version and add to changelog.

File: tests/tests_integration/test_contextualization/test_pnid_parsing.py (hunk 1 above)
Comment (chingyu-cognite) [reply tests/tests_integration/test_contextualization/test_pnid_parsing.py:8]: yes

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,15 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [0.20.2] - 2020-09-09
```
Comment (rebwib) [inline CHANGELOG.md:15]: Since this is a breaking change we should bump to version 0.21.0, right @sanderland

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,15 @@ Changes are grouped as follows
...
+### Removed
+- Removed `/parse` in Pnid Parser
+### Added
+- Added `/convert` in Pnid Parser
+### Fixed
```
Comment (rebwib) [inline CHANGELOG.md:20]: You don't need to include this, `fixed` is ment for bug fixes.

File: CHANGELOG.md (hunk 2 above)
Comment (chingyu-cognite) [reply CHANGELOG.md:15]: ah I bumped to 0.21.0, but forgot to change the CHANGELOG.md

Comment (rebwib) [reply CHANGELOG.md:15]: I see below that you bumped to 0.21.0 in pyproject.toml, so just need to change this to the correct version.

**Review by rebwib — APPROVED** [review]
Remove fixed and update to correct version in changelog, other than that LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (chingyu-cognite) [reply CHANGELOG.md]: ok

**Pull request merged**